plugins/brightness/PageComponent.qml

```diff
@@ -79,12 +79,11 @@ ItemPage {
         Column {
             anchors.left: parent.left
             anchors.right: parent.right
-
-            ListItem.Standard {
+            
```

Comment:
This line only changes whitespace, the whitespace can be removed

Reply:
hm, maybe a stray space? it removed and added the same